tangoObjects.py

```diff
@@ -279,15 +284,16 @@ def set(self, id, obj):
         return str(id)
 
     def get(self, id):
-        if str(id) in self.r.hkeys(self.hash_name):
+        if self.r.hexists(self.hash_name, str(id)):
             unpickled_obj = self.r.hget(self.hash_name, str(id))
             obj = pickle.loads(unpickled_obj)
             return obj
         else:
             return None
 
     def keys(self):
-        return self.r.hkeys(self.hash_name)
+        keys = map(lambda key : key.decode(), self.r.hkeys(self.hash_name))
+        return list(keys)
```

Comment:
It seems to be returning a list when keys are called on the TangoRemoteDictionary.